### PR TITLE
Highlights newly created module

### LINKED ISSUE #1510: Highlight newly created module in the Dashboard page
Sometimes it can be hard to see the newly created module, especially if you have lots of default named modules (or your sort method is set to alphabetical).

Suggest a highlight to indicate the newly created module - here's a mockup:

image

Originally posted in #1434

### PR DESCRIPTION
**Pull request by jpeterson976**
Fixes #1510 

Grabs the draftId of the newly created module and highlights it in the dashboard.

### CODE REVIEW

File: packages/app/obojobo-repository/shared/components/module.scss [hunk 1]
```diff
@@ -37,6 +37,11 @@
 		background-color: $color-banner-bg;
 	}
 
+	&.is-new {
+		background-color: lighten($color-action-minor, 15%);
+		font-weight: bold;
```
Comment (ctcuff) [inline packages/app/obojobo-repository/shared/components/module.scss:42]: Looks like setting the font-weight here also affects the font-weight of the buttons in the module's menu

**Review by ctcuff — APPROVED** [review]
Super minor CSS issue but looks good! Tested in Chrome, FF, and Safari.

**Review by zachberry — CHANGES_REQUESTED** [review]
This is awesome! Marking as 'request changes' based on what @ctcuff found - but once that css issue is resolved I'll approve

**Review by zachberry — APPROVED** [review]
(I merged in dev/16)

LGTM!

**Pull request merged**